### ISSUE TITLE
Update marshmallow to 2.18.1

### ISSUE THREAD

**Issue by ivan-artezio** [miLibris/flask-rest-jsonapi#165]
Marshmallow 2.18.0 raises ChangedInMarshmallow3Warning for nested schemas which is annoying when running tests and clogs the logs. Marshmallow should be updated to 2.18.1 where this issue was fixed.
https://marshmallow.readthedocs.io/en/3.0/changelog.html
https://github.com/marshmallow-code/marshmallow/pull/1136

**Comment by PATolk** [comment]
I needed to upgrade Marshmallow to 3.2.1
and now I get an error
  File "/lib/python3.6/site-packages/flask_rest_jsonapi/resource.py", line 131, in get
    result = schema.dump(objects).data
AttributeError: 'dict' object has no attribute 'data'

**Comment by TMiguelT** [comment]
Yes, the library has to be changed to make it Marshmallow 3 compatible. I've fixed this in my PR: #172. However it hasn't yet been merged.

**Comment by auvipy** [comment]
this can be closed now
